### ISSUE TITLE
ESI Shock Megademo crashes after intro

### ISSUE THREAD

**Issue by Haroldo64** [raydac/zxpoly#31]
Hi, ESI Shock Megademo crashes after intro

[SHOCK.zip](https://github.com/raydac/zxpoly/files/6389848/SHOCK.zip)

**Comment by raydac** [comment]
are you sure that it is a correctly tape version? does it work under another emulators?

**Comment by Haroldo64** [comment]
Yes, it run fine on Fuse

**Comment by Haroldo64** [comment]
https://worldofspectrum.org/archive/software/demos/shock-megademo-esi

**Comment by Haroldo64** [comment]
With last version I can see all demo in 48k mode. But everytime it has to load a new part I must rewind the tape manually, the tape does not stop by itself.
![Untitled 2](https://user-images.githubusercontent.com/53178600/162670532-c5df10ca-d995-4933-ac6b-2b79f6552c00.png)

**Comment by rossimariolee** [comment]
Hi, it does not work for me. There are some graphic bugs in part 2:
![Untitled 5](https://user-images.githubusercontent.com/41687618/162672113-d449be1a-db9a-4fd0-bf29-45c893d313a4.png)

**Comment by Haroldo64** [comment]
You are right, this part does not work as in Fuse emulator, there is a black band in the bottom too

**Comment by Alvarez81** [comment]
https://youtu.be/GIdZVtY4vPA?t=435

**Comment by Alessandro1970** [comment]
Hi,
It works quite good on my ZXPoly (windows and Mac M1 cpu)
Try to use Pentagon timing:
![Untitled 4](https://user-images.githubusercontent.com/34916641/163116602-774f069d-1a40-49d5-bd60-c4e648ef69da.png)

**Comment by PakkunKinoppi** [comment]
you are right, using Pentagon timings in the last ZxPoly pre-release it works well.
